sample/app/build.gradle

```diff
@@ -28,6 +28,9 @@ buildProperties {
     env {
         from System.getenv()
     }
+    cli {
+        from project.properties
```

Comment:
and "system properties" also include all the `System.getenv()` properties too? I mean, if I was using this in my project would it make more sense for me to do:

```groovy
buildProperties {
    project {
        from project.properties
    }
}
```

and this would encompass _all_ the non-custom properties files?